source/includes/mongodb-compatibility-table-node.rst

```diff
@@ -214,3 +228,4 @@
      - ✓
 
 .. [#unified-topology] When using Node.js Driver version 3.7, you must set the ``useUnifiedTopology`` flag to ``true`` for certain features.
+.. [#deprecated] Support for MongoDB 3.6 is deprecated in {+driver-short+} v6.9.
```

Comment:
S: perhaps this could have a more descriptive name to account for future notes that might be similar
```suggestion
.. [#server-3.6-deprecated] Support for MongoDB 3.6 is deprecated in {+driver-short+} v6.9.
```